Doc/library/functions.rst

```diff
@@ -631,21 +631,21 @@ are always available.  They are listed here in alphabetical order.
       Added the *closure* parameter.
 
 
-.. function:: filter(function, iterable)
+.. function:: filter(predicate, iterable)
```

Comment:
This will leave the docstring and the docs inconsistent with each other:

```
Help on class filter in module builtins:

class filter(object)
 |  filter(function, iterable, /)
```

Reply:
Sorry, 

I don't fully understand the structure of the repo.. (actually I don't know any language apart from `python`)

In what file should I also update the description? (if it is required at all of course)..

Reply:
I think the docs are fine as they are. This function has been a part of Python for years and the docs has not proved confusing for users. I'm sorry, but I suggest closing this PR.